scipy/stats/_binned_statistic.py

```diff
@@ -459,6 +459,43 @@ def binned_statistic_dd(sample, values, statistic='mean',
 
     .. versionadded:: 0.11.0
 
+    Examples
+    --------
+    >>> from scipy import stats
+    >>> import matplotlib.pyplot as plt
+    >>> from mpl_toolkits.mplot3d import Axes3D
+
+    Take 600 2-D arrays as an example. `binned_statistic_dd` can handle arrays
```

Comment:
This is phrased a bit unusually. The shape here is `(600, 2)`, which would be good to show at the end:
```
>>> data.shape
(600, 2)
```
And for phrasing maybe _"Take an array of 600 (x, y) coordinates as an example"_?